web/app/components/since.js

```diff
@@ -1,26 +1,29 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect } from "react";
 
-export default function Since( { props }) {
+export default function Since({ props }) {
   const [time, setTime] = useState(new Date());
-  const since = new Date(props);
+
 
   useEffect(() => {
     const timer = setInterval(() => {
       setTime(new Date());
-    }, 1000)
+    }, 1000);
     return () => {
-      clearInterval(timer)
-    }
-  }, [])
-  
+      clearInterval(timer);
+    };
+  }, []);
+
   // const daysSince = Math.floor((time - since) / (1000 * 24 * 60 * 60));
-  const secondsSince = Math.floor((time - since) / (1000));
-  
-  return ( 
+  const secondsSince = props
+    ? Math.floor((time - props) / 1000)
+    : "Never Eaten";
+
+  return (
+
     <div>
       {/* <p>{since ? `Days since last eaten: ${daysSince < 0 ? 0 : daysSince}` : 'Never eaten' }</p> */}
       {/* <p>{since ? `Seconds since last eaten: ${secondsSince < 0 ? 0 : secondsSince}` : 'Never eaten' }</p> */}
-      <p>{`Seconds since last eaten: ${secondsSince}`}</p>
+      <p>{`Seconds since last eaten: ${secondsSince < 0 ? 0 : secondsSince}`}</p>
```

Comment:
what's the use case for `secondsSince` being less than 0?

Reply:
I haven't double checked the math, but it ends up being -1 right after the button is pressed.

Reply:
ideally we fix whatever is going on with the math there then if this is just a workaround for a bug that can be addressed

Reply:
I've fiddled around with it a bit, but I haven't found a fix yet. I'll take a deeper dive into it when I rework the app to use a single timer.